demo/adminDashboard/demo.js

```diff
@@ -0,0 +1,46 @@
+(function (angular) {
+
+    angular.module('demo', ['znk.infra-web-app.adminDashboard'])
+        .config([
+            '$stateProvider',
+            function ($stateProvider) {
+                $stateProvider
+                    .state('eslink', {
+                        url: '/eslink',
+                        templateUrl: '/templates/esLink.template.html',
+                        controllerAs: 'vm'
+                    })
+                    .state('emetadata', {
+                        url: '/emetadata',
+                        templateUrl: '/templates/eMetadata.template.html',
+                        controllerAs: 'vm'
+                    });
+            }
+        ])
+        .constant('ENV', {
+            firebaseAppScopeName: "act_dashboard",
+            fbGlobalEndPoint: "https://znk-dev.firebaseio.com/",
+            fbDataEndPoint: "https://act-dev.firebaseio.com/",
+            appContext: 'student',
+            studentAppName: 'sat_app',
+            dashboardAppName: 'sat_dashboard',
+            videosEndPoint: "//dfz02hjbsqn5e.cloudfront.net/sat_app/",
+            mediaEndPoint: "//dfz02hjbsqn5e.cloudfront.net/",
+            backendEndpoint: "https://znk-web-backend-dev.azurewebsites.net/",
+            teachworksDataUrl: 'teachworks',
+            debug: true,
+            "elasticSearch": {
+                "host": "znk-elastic-dev4891.cloudapp.net:9200",
+                "apiVersion": "5.x",
```

Comment:
by default, it uses the latest version, but I wanted to use the same major version for non-breaking changes + for readability.